build.sbt

```diff
@@ -1080,13 +1080,36 @@ def initCommands(additionalImports: String*) =
 // This won't actually release unless on Travis.
 addCommandAlias("ci", ";clean ;release with-defaults")
 
-// OrganizeImports needs to run separately to clean up after the other rules
+lazy val enableFatalWarnings: String =
+  """set ThisBuild / scalacOptions += "-Xfatal-warnings""""
+
+lazy val disableFatalWarnings: String =
+  """set ThisBuild / scalacOptions -= "-Xfatal-warnings""""
+
 addCommandAlias(
   "quicklint",
-  ";scalafixAll --triggered ;scalafixAll ;scalafmtAll ;scalafmtSbt",
+  List(
+    enableFatalWarnings,
+    "scalafixAll --triggered",
+    "scalafixAll",
+    "scalafmtAll",
+    "scalafmtSbt",
+    disableFatalWarnings,
+  ).mkString(" ;"),
```

Comment:
Does this work like? I thought that sbt command alias need to start with `; ` but maybe I am wrong. See https://github.com/typelevel/sbt-typelevel/blob/3936eba4f2cdffafa13678517630e9880d727150/kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala#L47

Reply:
It's easy to check
```scala
scala> List("foo", "bar", "baz").mkString(" ;")
res1: String = foo ;bar ;baz
```
That seems like what we need. And yeah, I checked manually that `lint`/`quicklint` works as expected :)

Reply:
A leading semicolon used to be required but no longer is. Now that it's not required, I feel like it's more elegant to have whitespace after rather than before the semicolon.

Reply:
Huh, can't keep up :) thanks!